zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/K8sStandardInterpreterLauncher.java

```diff
@@ -155,7 +155,7 @@ public InterpreterClient launch(InterpreterLaunchContext context) throws IOExcep
             properties,
             buildEnvFromProperties(context),
             getZeppelinService(),
-            getZeppelinServiceRpcPort(),
+            Integer.parseInt(getZeppelinServiceRpcPort()),
```

Comment:
Maybe we should change the return value of [getZeppelinServiceRpcPort()](https://github.com/apache/zeppelin/blob/261eb395cdb3e94db5fc530ee08a89f1f37b51bf/zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/K8sStandardInterpreterLauncher.java#L119) to int.

Reply:
Make sense, let me update it.